src/som/interpreter/actors/EventualMessage.java

```diff
@@ -323,24 +331,40 @@ public static EventualMessage getCurrentExecutingMessage() {
 
   /**
    * Indicates that execution should stop and yield to the debugger,
-   * before the computed value is used to resolve the promise.
+   * before the message is processed.
+   */
+  public boolean haltOnReceive() {
+    return haltOnReceive;
```

Comment:
for me is confusing when reading the code, I have not seen this in Java before, aren't we following Java conventions? maybe `isEnabledHaltOnReceive` is better English?